pom.xml

```diff
@@ -96,9 +96,9 @@
         <hbase.guava.version>12.0.1</hbase.guava.version>
         <cassandra.version>3.0.0</cassandra.version>
         <cassandra.guava.version>18.0</cassandra.guava.version>
-        <kafka.version>1.0.1</kafka.version> <!-- also supports: 0.10.2.1, 0.9.0.1 -->
-        <zkclient.version>0.10</zkclient.version> <!-- corresponding versions: 0.10, 0.7 -->
-        <kafka.jopt.version>5.0.4</kafka.jopt.version> <!-- corresponding versions: 4.9, 3.2 -->
+        <kafka.version>2.0.0</kafka.version> <!-- also supports: 1.1.1, 1.0.2, 0.10.2.1, 0.9.0.1 -->
```

Comment:
This suggests that versions 1.1.1 - 2.0.0 are supported.

Reply:
is the comment unclear? it's meant to indicate that 2.0.0, 1.1.1, 1.0.2, 0.10.2.1 or 0.9.0.1 are supported

Reply:
I can't read.  The lists are in the opposite orders:).